packages/create-block/package.json

```diff
@@ -1,45 +1,45 @@
 {
-  "name": "@wordpress/create-block",
-  "version": "0.5.1",
-  "description": "Generates PHP, JS and CSS code for registering a block for a WordPress plugin.",
-  "author": "The WordPress Contributors",
-  "license": "GPL-2.0-or-later",
-  "keywords": [
-    "wordpress",
-    "block",
-    "scaffold"
-  ],
-  "homepage": "https://github.com/WordPress/gutenberg/tree/master/packages/create-block/README.md",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/WordPress/gutenberg.git",
-    "directory": "packages/create-block"
-  },
-  "bugs": {
-    "url": "https://github.com/WordPress/gutenberg/issues"
-  },
-  "engines": {
-    "node": ">=10.0",
-    "npm": ">=6.1"
-  },
-  "files": [
-    "lib"
-  ],
-  "main": "index.js",
-  "bin": {
-    "wp-create-block": "./index.js"
-  },
-  "dependencies": {
-    "chalk": "^2.4.2",
-    "commander": "^4.1.0",
-    "execa": "^4.0.0",
-    "inquirer": "^7.0.3",
-    "make-dir": "^3.0.0",
-    "mustache": "^4.0.0",
-    "pkg-install": "^1.0.0",
-    "write-pkg": "^4.0.0"
-  },
-  "publishConfig": {
-    "access": "public"
-  }
+	"name": "@wordpress/create-block",
```

Comment:
@ntwb - something to consider for the linter maybe?

Reply:
There is the [valid-values-name-scope](https://npmpackagejsonlint.org/docs/en/rules/valid-values/valid-values-name-scope) rule to define _allowed scope_, in this case we would want `@wordpress`

Reply:
Ahhh, I see the issue now, indentation, I didn't notice this when looking at just the comment in isolation, this was discussed in https://github.com/tclindner/npm-package-json-lint/issues/81, but at the time there were no great JSON AST parsers, I suspect that has not changed recently

Reply:
Thanks @ntwb for getting back. It seems like copy and paste issue then. I guess it means it was copy and paste issue, I better double check .editorconfig for JSON files in the templates 👍